kube_dns/metadata.csv

```diff
@@ -0,0 +1,8 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+kubedns.response_size.bytes.sum,gauge,,byte,,Size of the returns response in bytes.,0,kube_dns,response size sum
+kubedns.response_size.bytes.count,gauge,,response,,Number of responses on which the kubedns.response_size.bytes.sum metric is evaluated.,0,kube_dns,response size sample count
+kubedns.request_duration.seconds.sum,gauge,,request,second,Time (in seconds) each request took to resolve.,0,kube_dns,request duration time
```

Comment:
Orientation should be `-1`, not `0`. Orientation is about what value we think is good for the metric and it goes like this:
```
- lower value is better --> -1
- higher value is better --> 1
- there's no better value --> 0
```